README.md

```diff
@@ -1 +1,58 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
```

Comment:
- [OPTIONAL] Please kindly update your project title, as this will make your readme file more informative and professional.